react/hooks/useIsPriceAsync.js

```diff
@@ -0,0 +1,11 @@
+import { useSearchPage } from 'vtex.search-page-context/SearchPageContext'
+
+const useIsPriceAsync = () => {
+  const { searchQuery } = useSearchPage()
```

Comment:
Can this destructure break if it's outside of search? I mean, if useSearchPage() returns undefined, what happens?

Reply:
`useSearchPage` return `{}` when there is no context
anyway, I changed my code to be extra careful